README.md

```diff
@@ -171,13 +171,13 @@ JavaScript applications. It runs both in Node.js and browser.
 
 | Param | Type | Default |
 | --- | --- | --- |
-| [options] | <code>object</code> |  | 
-| [options.baseUrl] | <code>string</code> | <code>&quot;https://api.apify.com&quot;</code> | 
-| [options.maxRetries] | <code>number</code> | <code>8</code> | 
-| [options.minDelayBetweenRetriesMillis] | <code>number</code> | <code>500</code> | 
-| [options.requestInterceptors] | <code>Array.&lt;function()&gt;</code> |  | 
-| [options.timeoutSecs] | <code>number</code> |  | 
-| [options.token] | <code>string</code> |  | 
+| [options] | <code>object</code> |  |
+| [options.baseUrl] | <code>string</code> | <code>&quot;https://api.apify.com&quot;</code> |
+| [options.maxRetries] | <code>number</code> | <code>8</code> |
+| [options.minDelayBetweenRetriesMillis] | <code>number</code> | <code>500</code> |
+| [options.requestInterceptors] | <code>Array.&lt;function()&gt;</code> |  |
+| [options.timeoutSecs] | <code>number</code> |  |
+| [options.token] | <code>string</code> |  |
```

Comment:
What's up with all the whitespace?

Reply:
I think my eslint config automatically removes unnecessary whitespace.. is it meant to be there?

Reply:
I guess it does not matter functionally, but I when someone generates it next time, it'll add the whitespace again and we'll have all the changes visible  in the PR again, so I'd prefer if it stayed the same.

Reply:
@mnmkng I see.. I generated the Readme via the npm script.. should be fixed now